src/app/recipes/recipe-edit/recipe-edit.component.ts

```diff
@@ -74,14 +77,11 @@ export class RecipeEditComponent implements OnInit, OnDestroy {
     let recipeIngredients = new FormArray([]);
 
     if (this.editMode) {
-      
       this.recipeSub = this.store
         .pipe(
           select('recipes'),
           map((recipeState) => {
-            return recipeState.recipes.find(
-              (recipe, index) => index === this.id
-            );
+            return recipeState.recipes.find((recipe) => recipe._id === this.id);
```

Comment:
Create specific selector for specific `id` param

Reply:
added